### PR TITLE
chore: add volunteer TuhinBar

### PR DESCRIPTION
**Pull request by TuhinBar**
Want to join askdevs as a volunteer to help new devs to get started.

### CODE REVIEW

File: data/users/TuhinBar.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "name": "Tuhin Bar",
+  "bio": "Web Developer",
+  "image": "https://user-images.githubusercontent.com/85868593/206908263-e79c6354-ee85-40fe-b126-ce470b96e964.png",
```
Comment (codewithvoid) [inline data/users/TuhinBar.json:4]: use simpler url https://avatars.githubusercontent.com/u/85868593?v=4

File: data/users/TuhinBar.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  "categories": [
+    {
+      "slug": "web-development",
+      "topics": ["Programming","Full Stack development","MERN stack"]
```
Comment (codewithvoid) [inline data/users/TuhinBar.json:8]: programming is separate category, you can mention that with programming language.

E.g 

```json
    {
      "slug": "programming",
      "topics": ["python", "javascript"]
    }
```

Comment (codewithvoid) [inline data/users/TuhinBar.json:8]: nit: mention `Full Stack Developer` in bio?

Comment (codewithvoid) [inline data/users/TuhinBar.json:8]: just MERN would suffice here

**Comment by codewithvoid** [comment]
Thanks @TuhinBar for volunteering with AskDevs :)

Added some small comments. PTAL.

File: data/users/TuhinBar.json (hunk 2 above)
Comment (TuhinBar) [reply data/users/TuhinBar.json]: Changed bio and specified about MERN and Full stack development

**Comment by TuhinBar** [comment]
@codewithvoid made the changes accordingly. Please check it out.

**Pull request closed without merging**